### DISCUSSION TITLE
ETA for v0.14.0

### DISCUSSION THREAD

**Discussion by nojaf** [dotnet/BenchmarkDotNet#2521] [Q&A] [answered]
Hi, a recent fix made it into master which I'm interested in.
What is the timeframe for the next release, please?

**Comment by AndreyAkinshin** [comment]
@nojaf, typically I aim to publish releases monthly (approximately, the first week of the month). However, I failed to do it in February.

The next version is v0.14.0 and it should include some important updates and breaking changes. @timcassell made some awesome PRs that we want to include in the upcoming release. The changes are not so trivial, so I want to carefully review them, which is a time-consuming activity. Unfortunately, last month was quite busy for me and I didn't have enough time to do this. I really hope to finish all the reviews in February, properly test the changes, and publish v0.14.0 in the first week of March. Sorry for the inconvenience. The current scope of unresolved issues and unmerged PRs is available [here](https://github.com/dotnet/BenchmarkDotNet/milestone/47). cc @adamsitnik @timcassell

Meanwhile, you can use the [nightly versions](https://benchmarkdotnet.org/articles/guides/nuget.html#nightly) of BenchmarkDotNet.

P.S. Really appreciate your status update request (it helps to bump the priority of this activity in my to-do list), but please use Discussions instead of Issues for further questions.

**Reply by nojaf** [reply]
Hi Andrey, thank you for this feedback. The nightly feed is working for me.

As for the PS, nothing really guided me to raise a discussion instead of creating an issue.
The issue template is blank for this repository and the README does not mention questions via discussions.

**Reply by AndreyAkinshin** [reply]
@nojaf thanks for the note. Indeed, it would be better to add some guidelines and/or issue templates.

**Comment by eNeRGy164** [comment]
Small bump, as it is July by now 😉

**Comment by AndreyAkinshin** [comment, marked as answer]
Sorry for such a delay in our release schedule, my bad. This year was challenging for me in terms of free time I can allocate for open source. Meanwhile, @timcassell and @adamsitnik did a great job in processing issues and PRs and contributing the new awesome stuff. Right now, I'm reviewing all the discussions and commits that I missed. Hope to publish a new release in the span of two weeks. Once it's ready, I will also publish a status update with our future plans.
